pkg/image/binary/cluster.go

```diff
@@ -32,39 +32,28 @@ import (
 type ClusterService struct {
 }
 
-func (d *ClusterService) Create(name string, image string) (*types.ClusterManifest, error) {
-	data := exec.BashEval("buildah containers --json")
-	infos, err := listContainer(data)
-	if err != nil {
+func (s *ClusterService) Create(name string, image string) (*types.ClusterManifest, error) {
+	if err := s.Delete(name); err != nil {
 		return nil, err
 	}
 
-	for _, info := range infos {
-		if info.Containername == name {
-			cmd := fmt.Sprintf("buildah unmount %s && buildah rm  %s", info.Containername, info.Containername)
-			if err = exec.Cmd("bash", "-c", cmd); err != nil {
-				return nil, err
-			}
-		}
-	}
-
-	cmd := fmt.Sprintf(" buildah from --pull=never --name %s %s && buildah mount %s ", name, image, name)
-	err = exec.Cmd("bash", "-c", cmd)
+	cmd := fmt.Sprintf("buildah from --pull=never --name %s %s && buildah mount %s ", name, image, name)
+	err := exec.Cmd("bash", "-c", cmd)
 	if err != nil {
 		return nil, err
 	}
```

Comment:
```suggestion
	if err := exec.Cmd("bash", "-c", cmd); err != nil {
		return nil, err
	}
```
this can avoid `err` reassign.